doc/source/changelog.rst

```diff
@@ -1,5 +1,9 @@
 Changelog
 ******************************
+- **v0.10.0** (*Unreleased*):
```

Comment:
Do we need to to continue maintain 2 changelogs?

Reply:
no. we have never maintained 2 changelogs. we are switching to mkdocs with this release.

Reply:
Then can `doc` be removed?

Reply:
Yeah sorry, above comment should've been: "do we need to _begin_ maintaining 2 changelogs"

Reply:
yeah `doc` is toast. :toilet:

Reply:
good riddance to sphinx...